packages/wallet/src/test-utils/launchNode.ts

```diff
@@ -140,20 +170,34 @@ export const launchNode = async ({
       chainConfigPathToUse = tempChainConfigFilePath;
     }
 
-    const child = spawn(command, [
-      'run',
-      '--ip',
-      ipToUse,
-      '--port',
-      portToUse,
-      '--db-type',
-      'in-memory',
-      '--consensus-key',
-      consensusKey,
-      '--chain',
-      chainConfigPathToUse as string,
-      ...args,
-    ]);
+    const child = spawn(
+      command,
+      [
+        'run',
+        ['--ip', ipToUse],
+        ['--port', portToUse],
+        useInMemoryDb ? ['--db-type', 'in-memory'] : ['--db-path', tempDirPath],
+        ['--min-gas-price', '0'],
+        poaInstant ? ['--poa-instant', 'true'] : [],
+        ['--consensus-key', consensusKey],
+        ['--chain', chainConfigPathToUse as string],
```

Comment:
To avoid using this `chainConfigPathToUse as string` cast multiple times, you could:

```ts
let chainConfigPathToUse: string;

if (chainConfigPath) {
  chainConfigPathToUse = chainConfigPath;
} else {
  // ...
  chainConfigPathToUse = tempChainConfigFilePath
}
```